packages/hint-compat-api/src/css.ts

```diff
@@ -223,6 +237,37 @@ const walk = (ast: ContainerBase, context: Context) => {
     reportUnsupported(reportsMap, context);
 };
 
+export const formatAlternates = (language: string, unsupported: UnsupportedBrowsers, formatFeature?: (name: string) => string): string[] => {
+    const groupedAlternates = new Map<string, BrowserAlternateDetails[]>();
```

Comment:
There's a lot going on in `formatAlternates` 😵
What do you think of splitting it into 2 more functions: 

* `groupAlternates`
* `createAlternateMessages`

And add some documentation with an example?

Reply:
Looking at this, I think just splitting out a separate `groupAlternatives` function for that part of the work helps enough. Splitting both out would make this function largely pass-through which I don't think is needed.